### PR TITLE
Create document-query-command-supported

### PR DESCRIPTION
**Pull request by dandv**
Based on https://developer.mozilla.org/en-US/docs/Web/API/Document/queryCommandSupported and testing at CBT

### CODE REVIEW

**Comment by cvrebert** [comment]
@dandv Travis JSON validation check is failing here too; you need to add https://github.com/Fyrd/caniuse/blob/ca56a2a7cc20ecdfb6a53fe377224e4a75f264ae/sample-data.json#L223

File: features-json/document-query-command-supported.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
+{
+  "title":"Document.queryCommandSupported()",
+  "description":"Reports whether or not the specified editor query command is supported by the browser.",
+  "spec":"https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#querycommandsupported()",
+  "status":"wd",
```
Comment (cvrebert) [inline features-json/document-query-command-supported.json]: It's an Editor's Draft, not a Working Draft

File: features-json/document-query-command-supported.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    "and_uc":{
+      "9.9":"u"
+    }
+  },
+  "notes":"",
+  "notes_by_num":{
+    "1": "Before Firefox 41, for the command 'paste' Firefox incorrectly returned true when the paste feature was available but the calling script had insufficient privileges to actually perform the action. Also, 'cut' and 'copy' were returning false. Now they return true in user-initiated or privileged code."
+  },
+  "ucprefix":false,
+  "parent":"Document.execCommand()",
```
Comment (cvrebert) [inline features-json/document-query-command-supported.json]: `parent` values should refer to filenames (without the `.json` suffix)

File: features-json/document-query-command-supported.json [hunk 3]
```diff
@@ -0,0 +1,227 @@
+{
+  "title":"Document.queryCommandSupported()",
+  "description":"Reports whether or not the specified editor query command is supported by the browser.",
+  "spec":"https://dvcs.w3.org/hg/editing/raw-file/tip/editing.html#querycommandsupported()",
+  "status":"ed",
```
Comment (dandv) [inline features-json/document-query-command-supported.json]: So `ed` doesn't stand for Editor's Draft. What should be put for that value? Maybe worth mentioning that in the contrib guide.

Comment (Fyrd) [inline features-json/document-query-command-supported.json]: @dandv You'll want `unoff` there. Just updated the contributions file to clarify.

**Comment by Fyrd** [comment]
Thanks!

**Pull request merged**